src/components/FilterComponent/index.tsx

```diff
@@ -20,6 +20,38 @@ export const FilterComponent: React.SFC<FilterComponentProps> = ({
   const cleanParams = sanitizeParams(params, options);
   const filters: CSEAdvancedFilterModels = createFilters(options, cleanParams);
 
+  const sortFilters = () => {
+    if (filters.claimFilter) {
+      filters.claimFilter.filterOptions.sort((a, b) => {
+        const x = a.key.toLowerCase();
+        const y = b.key.toLowerCase();
+        if (x < y) {
+          return -1;
+        }
+        if (x > y) {
+          return 1;
+        }
+
+        return 0;
+      });
+    }
+    if (filters.targetFilter) {
+      filters.targetFilter.filterOptions.sort((a, b) => {
+        const x = a.key.toLowerCase();
+        const y = b.key.toLowerCase();
+        if (x < y) {
+          return -1;
+        }
+        if (x > y) {
+          return 1;
+        }
+
+        return 0;
+      });
+    }
+  };
+  sortFilters();
+
```

Comment:
Consider breaking these out into their own helper functions to avoid making this code to complex.
Additionally this will solve the code climate issues.